shared/src/main/scala/doodle/core/Gradient.scala

```diff
@@ -0,0 +1,25 @@
+package doodle.core
+
+sealed trait Gradient extends Product with Serializable {}
+
+object Gradient {
+  final case class Linear(start: Point, end: Point, stops: Seq[(Color, Double)], cycleMethod: CycleMethod) extends Gradient
+  final case class Radial(outer: Point, inner: Point, radius: Double, stops: Seq[(Color, Double)], cycleMethod: CycleMethod) extends Gradient
+
+  def DichromaticVertical(color1: Color, color2: Color, length: Double) =
```

Comment:
For consistency with the rest of the code base this method name, and those below, should start with a lowercase letter.